simplq/src/components/pages/Admin/QueueSettings.jsx

```diff
@@ -38,14 +43,26 @@ export default ({ queueId }) => {
 
   const toggleModal = () => setIsModalOpen((isOpen) => !isOpen);
 
+  const handleSelfJoinCb = React.useCallback(
```

Comment:
let's use Checkbox instead of Cb here and in other places.

Reply:
Sure

Reply:
Done